test/object_ref.jl

```diff
@@ -69,4 +72,17 @@ end
         yield()
         @test local_count(oid) == 0
     end
+
+    @testset "Task argument object ref" begin
+        obj = Ray.put(1)
+        # XXX: The "task dependency" ref count goes to 0 when the task is
+        # complete; Ray tests use an actor task which can be triggered to
+        # complete by sending a "signal" (via another task call) to avoid race
+        # conditions.  Here we're just hoping that the following test runs
+        # within ~3s.
+        ret_obj = Ray.submit_task(x -> (Sleep(3); Ray.get(x)), (obj,))
```

Comment:
Why isn't this failing?
```suggestion
        ret_obj = Ray.submit_task(x -> (sleep(3); Ray.get(x)), (obj,))
```

Reply:
😬